model/book.rb

```diff
@@ -8,6 +8,16 @@ def initialize(publisher, cover_state, publish_date, archived: false)
     @cover_state = cover_state
   end
 
```

Comment:
for the Entity class to work and parse your JSON file you have to implement this method
`def self.constructor_pattern
    %w[name id items]
  end`